.travis.yml

```diff
@@ -90,21 +90,73 @@ install:
   - export AWESOME_VERSION="${TRAVIS_BRANCH}-g$(git rev-parse --short HEAD)"
   - 'if [ "$TRAVIS_PULL_REQUEST" != false ]; then AWESOME_VERSION="${AWESOME_VERSION}-PR${TRAVIS_PULL_REQUEST}"; fi'
 
+  # Functions for custom Travis folds.
+  - |
+    travis_fold_start() {
+        echo -n "travis_"
+        echo "fold:start:$1"
+        travis_fold_current="$1"
+    }
+    travis_fold_end() {
+        echo -n "travis_"
+        echo "fold:end:$travis_fold_current"
+    }
```

Comment:
I've gone that route, but it seemed like it cannot be (sanely) used everywhere, so I've reverted it.
It makes sense for the simple `make` though, of course.  We can keep that in mind.